bin/build-openapi-yaml.rb

```diff
@@ -576,12 +578,24 @@ def build_openapi_paramobj(jsonparamobj, paramtype)
   paramobj = {}
   paramobj["name"] = jsonparamobj["name"]
   paramobj["in"] = paramtype
-  paramobj["schema"] = {}
-  paramobj["schema"]["type"] = "string"
+  # Cover Java generics here
+  paramobj["schema"] = if %w[Collection<String> List<String>].include? jsonparamobj['javaType']
```

Comment:
Set?

Reply:
added this (although don't think anything uses it yet)